app/account/forms.py

```diff
@@ -6,7 +6,7 @@
     SubmitField
 )
 from wtforms.fields.html5 import EmailField
-from wtforms.validators import InputRequired, Length, Email, EqualTo
+from wtforms.validators import InputRequired, DataRequired, Length, Email, EqualTo
```

Comment:
Data required simply ensures that data exists in the field I believe, as opposed to input. because we populate the fields with default csv values, I believe that it makes sense to simply ensure that data exists in the forms fields, as opposed to checking for new input, as sometimes they may want to simply ignore inputting the default value again.